### PR TITLE
Enhancement/skale 1747 Fix build and deploy

### PR DESCRIPTION
**Pull request by alexgex**

### CODE REVIEW

File: bounty/build.sh [hunk 1]
```diff
@@ -1,24 +1,28 @@
 #!/usr/bin/env bash
 
-ver=0.3.0
+# Use for manual building/publishing of bounty-agent docker image
+# Use "p" parameter to publish after build
 
-echo "Building SKALE Bounty agent v${ver}..."
-docker build -t skalelabshub/bounty-agent:${ver} .. -f ../Dockerfile.bounty &&
-echo "Bounty agent v${ver} was built successfully" &&
+VERSION=$(cat VERSION)-test
```
Comment (dmytrotkk) [inline bounty/build.sh]: why `-test` in version?

File: bounty/build.sh [hunk 2]
```diff
@@ -1,24 +1,28 @@
...
-    echo "Pushing Bounty agent v${ver}" &&
-    docker push skalelabshub/bounty-agent:${ver} &&
+    docker tag skalelabshub/bounty-agent:${VERSION} skalelabshub/bounty-agent:latest &&
+    echo "Bounty agent v${VERSION} was tagged to the latest successfully" &&
+    echo "$LINE"
+    echo "Pushing Bounty agent v${VERSION}" &&
+    docker push skalelabshub/bounty-agent:${VERSION} &&
+    echo "$LINE"
+    echo "Bounty agent v${VERSION} was pushed successfully" &&
     echo "-----------------------------" &&
```
Comment (dmytrotkk) [inline bounty/build.sh]: mb remove this?

**Review by dmytrotkk — DISMISSED** [review]

File: bounty/build.sh (hunk 2 above)
Comment (alexgex) [reply bounty/build.sh]: yes, thanks, just miss it)

File: bounty/build.sh (hunk 1 above)
Comment (alexgex) [reply bounty/build.sh]: it was simpler)
corrected

**Review by dmytrotkk — APPROVED** [review]

**Pull request merged**